Expectations: 1234
Expectations2: 
Errors: 56
Recursion: 7
Skips: SSS
Warnings: WWW
Output: 
Backtraces: 89abc
End: 

══ Skipped ═════════════════════════════════════════════════════════════════════
1. Skip:1 (@tests.R#62) - Reason: skip

2. Skip:2 (@tests.R#69) - Reason: skip

3. Skip:3 (@tests.R#72) - Reason: empty test

══ Warnings ════════════════════════════════════════════════════════════════════
1. Warning:1 (@tests.R#78) - abc

2. Warning:2 (@tests.R#84) - def

3. Warning:2 (@tests.R#85) - ghi

══ Failed ══════════════════════════════════════════════════════════════════════
── 1. Failure: Failure:1 (@tests.R#12)  ────────────────────────────────────────
Failure has been forced

── 2. Failure: Failure:2a (@tests.R#16)  ───────────────────────────────────────
Failure has been forced

── 3. Failure: Failure:2b (@tests.R#19)  ───────────────────────────────────────
FALSE isn't true.

── 4. Failure: Failure:loop (@tests.R#24)  ─────────────────────────────────────
`i` not equal to 2.
1/1 mismatches
[1] 1 - 2 == -1

── 5. Error: Error:1 (@tests.R#33)  ────────────────────────────────────────────
stop
Backtrace:
 1. NULL
 2. base::eval(code, test_env)

── 6. Error: Error:3 (@tests.R#47)  ────────────────────────────────────────────
!
Backtrace:
 1. f()
 2. g()
 3. h()

── 7. Error: Recursion:1 (@tests.R#56)  ────────────────────────────────────────
This is deep
Backtrace:
  1. f(25)
  2. f(x - 1)
  3. f(x - 1)
  4. f(x - 1)
  5. f(x - 1)
  6. f(x - 1)
  7. f(x - 1)
  8. f(x - 1)
  9. f(x - 1)
 10. f(x - 1)
 11. f(x - 1)
 12. f(x - 1)
 13. f(x - 1)
 14. f(x - 1)
 15. f(x - 1)
 16. f(x - 1)
 17. f(x - 1)
 18. f(x - 1)
 19. f(x - 1)
 20. f(x - 1)
 21. f(x - 1)
 22. f(x - 1)
 23. f(x - 1)
 24. f(x - 1)
 25. f(x - 1)
 26. f(x - 1)

── 8. Error: errors thrown at block level are entraced (@tests.R#99)  ──────────
foo
Backtrace:
 1. f()
 2. g()

── 9. Error: errors thrown from a quasi-labelled argument are entraced (@tests.R
foo
Backtrace:
 1. testthat::expect_is(foo(), "foo")
 4. foo()

── 10. Error: errors thrown from a quasi-labelled argument are entraced (deep ca
foo
Backtrace:
 1. testthat::expect_is(f(), "foo")
 4. f()
 5. g()
 9. foo()
  ... and 2 more


Maximum number of 10 failures reached, some test results may be missing.

══ DONE ════════════════════════════════════════════════════════════════════════
